.devcontainer/devcontainer.json

```diff
@@ -26,7 +26,8 @@
 				"icon": "terminal-bash"
 			}
 		},
-		"terminal.integrated.defaultProfile.linux": "bash"
+		"terminal.integrated.defaultProfile.linux": "bash",
+		"editor.renderWhitespace": "all"
```

Comment:
Change this setting to `"trailing"`

- See https://github.com/UBCSailbot/docs/pull/114 for an explanation